src/styles/blocks/block.scss

```diff
@@ -0,0 +1,52 @@
+.body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  width: $calendarWidth;
+
+  &__day {
+    width: $boxSize;
+    height: $boxSize;
+    background: #eee;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.5s ease-in-out;
+
+    &:hover ,
+    &:focus {
+      cursor: pointer;
+      background: pink;
+      transform: translateY(-20px);
+    }
+
+    @for $ind from 1 through 31 {
+      &:nth-child(#{$ind})::before {
+        content: "#{$ind}";
+        font-size: 30px;
+        font-family: Arial, Helvetica, sans-serif;
+      }
+    }
+  }
+}
+
+@each $key , $value in $margins {
+  .calendar--#{$key} .calendar--start-day-wed:first-child {
```

Comment:
remove hard-coded 'wed', use key variable instead to construct the right modifier class